commands/Transfering/transfer.js

```diff
@@ -36,8 +36,31 @@ function successMessage(res, amount){
   broadcastOperation(amount, user, transferred_to);
 }
 
+function canRemoveByAngryPoints(value) {
+  if (value > 0) {
+    return false
+  }
+  var angryPointsLimit = Libs.ResourcesLib.userRes("angryPointsLimit")
+  if (-value > angryPointsLimit.value()) {
+    return false
+  }
+
+  var angryPoints = Libs.ResourcesLib.userRes("angryPoints")
+  if (angryPoints.value() + value < 0) {
```

Comment:
angryPoints.value() - always negative after first remove! (see line 58 and 84)
value - always negative

this case <0 always!